command_plugins/github/plugin.py

```diff
@@ -321,6 +329,67 @@ def add_outside_collab_command(self, data, user_data, collab, org, repo, permiss
                                                     org, repo),
                      markdown=True, thread=data["ts"])
 
+    @hubcommander_command(
+        name="!SetRepoPermissions",
+        usage="!SetRepoPermissions <OrgWithRepo> <Repo> <Team> <Permission>",
+        description="This will set team permissions on a repository .",
+        required=[
+            dict(name="org", properties=dict(type=str, help="The organization that contains the repo."),
+                 validation_func=lookup_real_org, validation_func_kwargs={}),
+            dict(name="repo", properties=dict(type=str, help="The repository to add the team to."),
+                 validation_func=extract_repo_name, validation_func_kwargs={}),
+            dict(name="team", properties=dict(type=str, help="The team's name.")),
+            dict(name="permission", properties=dict(type=str.lower, help="The permission to grant, must be one "
+                                                                         "of: `{values}`"),
+                 choices="permitted_permissions")
+        ],
+        optional=[]
+    )
+    @auth()
+    @repo_must_exist()
+    @team_must_exist()
+    def set_team_permissions(self, data, user_data, team, org, repo, permission):
```

Comment:
Please rename to `set_repo_permissions_command`

Reply:
Done